src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -1339,6 +1359,10 @@ public static int MethodWithDefaultParameter(int x, int y = 10)
             return x + y;
         }
 
+        public int? MethodReturnsNullableInt(int a) => a > 0 ? (int?)a : null;
+
+        public int MethodAcceptsNullableArgs(int? a, int? b) => (a.HasValue ? 1 : 0) + (b.HasValue ? 1 : 0);
```

Comment:
nit: For this test method, I think it would be better to have:

```
public int MethodAcceptsNullableArgs(int? a, int? b) => (a.HasValue ? a.Value : 0) + (b.HasValue ? b.Value : 0);
```

That way you can further validate that you don't accidently have an issue where you're swapping the values of a & b.

Reply:
I considered that, but the problem is that then I can't discern between 0 and null being received by the server method. I very much wanted to assert that the server method can tell the difference between a `default(int)` and `default(int?)`.